### PR TITLE
Install script for custom geckodriver versions

### PR DESCRIPTION
**Pull request by joesiewert**

### CODE REVIEW

File: tests/packages/geckodriver.bats [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+setup() {
+  export GECKODRIVER_VERSION="0.16.1"
+}
+
+@test "[geckodriver.sh] Installs successfully" {
+  mkdir -p "${HOME}/cache"
```
Comment (mlocher) [inline tests/packages/geckodriver.bats]: Why do we need to create the cache directory here, and if we need, I'd add this to the `setup` function.

File: tests/packages/geckodriver.bats [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+setup() {
+  export GECKODRIVER_VERSION="0.16.1"
```
Comment (mlocher) [inline tests/packages/geckodriver.bats:4]: I would add a command to clear out the cached download, so we always install from the upstream provider and get notified if they change a URL or something similar.

**Review by mlocher — CHANGES_REQUESTED** [review]
Love the bats tests, can you make sure we call bats tests for packages on the build? Maybe in a separate pipeline?

Other than that only 2 smaller issues, which I'd like to get some feedback on.

File: tests/packages/geckodriver.bats (hunk 2 above)
Comment (joesiewert) [reply tests/packages/geckodriver.bats:4]: Don't we already clear the entire cache during setup in `tests/setup.sh`?

Comment (mlocher) [reply tests/packages/geckodriver.bats:4]: Yeah, I wanted to get rid of that, once we have all tests converted to bats though, and have each test take care of whether it wants to use the cache or not. 

I started to add that to all other bats files, e.g. https://github.com/codeship/scripts/blob/master/tests/cache/bower.bats#L4

**Review by mlocher — APPROVED** [review]
:shipit:

**Pull request merged**